### PR TITLE
Add config knob for faults in tenant subscriptions

### PR DESCRIPTION
**Pull request by sbagalur**
Add a config knob to enable/disable subscription of faults in tenants.

### CODE REVIEW

File: ciscoaci-puppet/ciscoaci/manifests/aim_config.pp [hunk 1]
```diff
@@ -25,6 +25,7 @@
   $mcast_ranges = '225.2.1.1:225.2.255.255',
   $multicast_address = '225.1.2.3',
   $gen1_hw_gratarps = 'False'
+  $enable_faults_subscriptions = 'False'
```
Comment (tbachman) [inline ciscoaci-puppet/ciscoaci/manifests/aim_config.pp:28]: The line above this needs a comma added

**Review by tbachman — CHANGES_REQUESTED** [review]
I think you also need to define the "enable_faults_subscription" parameter in the same puppet file (at the top - I think it's part of the class declaration).

File: ciscoaci-puppet/ciscoaci/manifests/aim_config.pp (hunk 1 above)
Comment (sbagalur) [reply ciscoaci-puppet/ciscoaci/manifests/aim_config.pp:28]: Added the comma. The line above is part of the class declaration. It is what is defining the parameter.

**Review by tbachman — APPROVED** [review]
Ah, good point - thanks!

**Pull request merged**